raw_data.php

```diff
@@ -693,6 +703,9 @@ function get_stops($swLat, $swLng, $neLat, $neLng, $tstamp = 0, $oSwLat = 0, $oS
 
 function get_gyms($swLat, $swLng, $neLat, $neLng, $tstamp = 0, $oSwLat = 0, $oSwLng = 0, $oNeLat = 0, $oNeLng = 0)
 {
+    if (!is_null($adapter)) {
```

Comment:
What does this exactly do? looks like if theres no value its gona return the following stated, but what is it exactly?
Thanks man

Reply:
$adapter is NULL when not running my fork.
It contains MonkeyFork object when running monkey fork. 
Check out line 8 in raw_data.php

Reply:
Ya on line 8 you defined get_gyms to be your class MonkeyFork, i understand now that what you did there is an alternative to not run your fork interesting, but if u have multiple forks how would you apply it? since we know theres more forks there.

Reply:
I don't care about the rest.
As far as $adapter is concerned, it runs from adapter if it exists, else, it runs from the rest of the code.

Notice the `return` on line 707

Reply:
ahhh, so if no $adapter = other code for the rest of forks works, i see. Smart thats what i was missing :P

Reply:
This way, I don't need to break anything in this raw_data file. No touching on messy parts as well.